application/lib/model/Output.dart

```diff
@@ -0,0 +1,12 @@
+class Output<T1> {
+  Output(this._output);
+  T1 _output;
+
+  void setOutput(T1 input) {
+    _output = input;
+  }
+
+  T1 getOutput() {
+    return _output;
+  }
+}
```

Comment:
Når vi kun har en type, så lad være med at skrive T1, så bare skriv T
Derudover hvad er formålet med denne klasse, ud over at output noget?
```suggestion
class Output<T> {
  Output(this._output);
  T _output;

  void setOutput(T input) {
    _output = input;
  }

  T getOutput() {
    return _output;
  }
}
```